tls/s2n_x509_validator.c

```diff
@@ -286,40 +286,40 @@ static uint8_t s2n_verify_host_information(struct s2n_x509_validator *validator,
     return verified;
 }
 
-S2N_RESULT s2n_x509_validator_validate_cert_chain(struct s2n_x509_validator *validator, struct s2n_connection *conn,
-        uint8_t *cert_chain_in, uint32_t cert_chain_len, s2n_pkey_type *pkey_type, struct s2n_pkey *public_key_out) {
+S2N_RESULT s2n_read_asn1_cert(struct s2n_stuffer *cert_chain_in_stuffer, struct s2n_blob *asn1_cert) {
```

Comment:
Nit: Can you either name methods starting with "s2n_x509_validator" to match the module, or make them static to indicate they're private, or both?

Reply:
I fixed this in 567385d3026256dc1fd2d91b4e892173158db3d4